### PR TITLE
Implement find_images_by_names with Pyxis GraphQL

### PR DESCRIPTION
**Pull request by ElenaKarolinaSemanova**
Implement find_images_by_names with Pyxis GraphQL with same projection as LightBlue's result, Jira: CWFHEALTH-1251

### CODE REVIEW

**Comment by ElenaKarolinaSemanova** [comment]
@qixiang @mulaieva Please review

File: freshmaker/pyxis_gql.py [hunk 1]
```diff
@@ -90,7 +90,10 @@ def _get_image_projection(self, include_rpms=True):
         projection = [
             ds.ContainerImage.architecture,
             ds.ContainerImage.brew.select(
+                ds.Brew.package,
```
Comment (qixiang) [inline freshmaker/pyxis_gql.py:93]: I realized that we have some fields included implicitly in LightBlue result, so when I say return same data as the LightBlue result, it's actually inaccurate, some of these fields are not used by any code at all (e.g. `ds.Brew.completion_date`), so could you remove these new fields in this PR? I'll check the existing code later to confirm whether these fields are used.

File: freshmaker/pyxis_gql.py [hunk 2]
```diff
@@ -90,7 +90,10 @@ def _get_image_projection(self, include_rpms=True):
...
+                ds.Brew.nvra,
```
Comment (qixiang) [inline freshmaker/pyxis_gql.py:94]:
```suggestion
```

File: freshmaker/pyxis_gql.py [hunk 3]
```diff
@@ -90,7 +90,10 @@ def _get_image_projection(self, include_rpms=True):
...
                 ds.Brew.build,
+                ds.Brew.completion_date,
```
Comment (qixiang) [inline freshmaker/pyxis_gql.py:96]:
```suggestion
```

File: tests/test_pyxis_gql.py [hunk 4]
```diff
@@ -319,3 +319,105 @@ def test_pyxis_graphql_find_images_by_nvr():
...
+            "total": 2,
+        }
+    }
+
+    pyxis_gql = PyxisGQL(
+        url="graphql.pyxis.local", cert=("/path/to/crt", "/path/to/key")
+    )
+    flexmock(Client).should_receive("execute").and_return(copy.deepcopy(result))
+
+    images = pyxis_gql.find_images_by_names("foo")
```
Comment (qixiang) [inline tests/test_pyxis_gql.py:422]: In the mocked data, the image name is `foobar-container` instead of `foo`.

**Comment by ElenaKarolinaSemanova** [comment]
@qixiang I resolved your suggestions, you can take a final look

**Review by qixiang — APPROVED** [review]
LGTM

**Pull request merged**